### DISCUSSION TITLE
trigger web hook when intended user offline with message payload

### DISCUSSION THREAD

**Discussion by muhun** [emqx/emqx#8526] [General]
we want to send notification when user offline , for example user A subscribed topic X, when user A offline , someone publish messages to topic X , then we need to trigger webhook , with message payload and user A client info . How can we do this ? is it possible to do with rule engine and sql ?

**Comment by zmstone** [comment]
hi
When a client is offline, the client information is gone.
I’m afraid there is no way to share this information with a webhook even if the client information can be kept.
there could be a more than one client subscribing to the topic
